packages/framework/composer.json

```diff
@@ -55,7 +53,7 @@
         "doctrine/doctrine-fixtures-bundle": "^3.0.2",
         "doctrine/persistence": "~1.2.0",
         "elasticsearch/elasticsearch": "^6.0.1",
-        "fp/jsformvalidator-bundle": "^1.5.1",
+        "fp/jsformvalidator-bundle": "dev-master",
```

Comment:
Beware, relying on `dev-master` is a bad practice. I spend an awful lot of time on removing these kind of dependencies (https://github.com/shopsys/shopsys/commit/27f3f016c43f187b8d129d42307c20948f620e3c#diff-06cf1370c3d18b48f45aa6b3056b638f, https://github.com/shopsys/shopsys/commit/cba4f0715ecc8ca3ef9491f6164722935a65a356#diff-06cf1370c3d18b48f45aa6b3056b638f, https://github.com/shopsys/shopsys/commit/fd62f2058938ce9caa8545735cd78fa02ca119f0#diff-06cf1370c3d18b48f45aa6b3056b638f) and I would like to discourage You from introducing `dev-master` dependency again.

If You need a newer version, maintainer usually releases such version if You ask.

Reply:
Hello, I know, I've been editing this package (form validator) a lot of much. I wanted to upgrade to dev-master, tested, (fixed) and then ask the owner to release a new version.